.travis.yml

```diff
@@ -19,5 +22,14 @@ before_cache:
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
 
+# For a PR TRAVIS_BRANCH is set to the target branch which is master.
+# We need to make sure it's not a PR when releasing.
 script:
-   - sbt ++$TRAVIS_SCALA_VERSION ciBuild
+  - if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ];
```

Comment:
I'll be taking it out as per the other comment, useful to know though this is my first travis-ing so def up for tips.